servant/servant.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               servant
-version:            0.20.2
+version:            0.20.3
```

Comment:
Probably better to wait with bumping until we do the release. Because it's reasonably likely that we'll merge e.g. two major changes that would both require major bumps.

Reply:
@ysangkok removed the version bump.